mule-gateway/modules/ROOT/pages/mule-gateway-encryption-mule4.adoc

```diff
@@ -74,9 +74,9 @@ wrapper.java.additional.100=-Danypoint.platform.client_id=![smzbg+aV2HbiC9Fv8KBs
 --
 
 [NOTE]
-The `![` prefix and  `]` suffix are required because they specify to the Mule Runtime Engine that the enclosed value is encrypted.
+The `![` prefix and  `]` suffix are required because they specify to Mule Runtime Engine that the enclosed value is encrypted.
 
-For the Mule runtime engine to be able to decrypt your secured values, you need to pass your secret key when starting your Mule runtime using the `anypoint.platform.encryption_key` property:
+When starting Mule runtime engine, you must pass your secret key, using the `anypoint.platform.encryption_key` property:
```

Comment:
I wonder if we can sync on whether to use Mule, Mule runtime engine, or Mule runtime for subsequent mentions.

Reply:
I think we should say "Mule runtime" on subsequent mentions, just because we say "Mule blahblah" about some other things which are not Mule runtime engine, and so it's parallel and *maybe* easier for translation?

I did not follow that in this set of changes. I can make another pass tomorrow or let it be.